maxfw/utils/image_utils.py

```diff
@@ -34,85 +118,143 @@ def __init__(self, remove_alpha_channel=False, grayscale=False, normalize=False,
         # Param sanity check
         assert self.standardize in [True, False]
         assert self.normalize in [True, False]
-        assert self.remove_alpha_channel in [True, False]
+        assert self.keep_alpha_channel in [True, False]
         assert self.verbose in [True, False]
-        assert type(self.to_file) == str
         assert int(normalize) + int(standardize) < 2, "Setting both normalize and stardize to True is not possible."
+        assert self.to_dtype in self.dtype_map.keys() or self.to_dtype is None, "The dtype should be either 'full', 'half', 'double', 'int' or 'uint8'."
```

Comment:
@splovyt `assert self.to_dtype in self.dtype_map or self.to_dtype is None` should suffice.
I'll edit it after merging.

Reply:
Sure! (just seeing this now.. woops)